gtdynamics/statics/StaticWrenchFactor.cpp

```diff
@@ -33,13 +33,13 @@ namespace gtdynamics {
 StaticWrenchFactor::StaticWrenchFactor(
     const std::vector<gtsam::Key> &wrench_keys, gtsam::Key pose_key,
     const gtsam::noiseModel::Base::shared_ptr &cost_model, double mass,
-    const boost::optional<gtsam::Vector3> &gravity)
+    const std::optional<gtsam::Vector3> &gravity)
     : Base(cost_model, wrench_keys), mass_(mass), gravity_(gravity) {
   keys_.push_back(pose_key);
 }
 
 Vector StaticWrenchFactor::unwhitenedError(
-    const Values &x, boost::optional<std::vector<Matrix> &> H) const {
+    const Values &x, gtsam::OptionalMatrixVecType H) const {
```

Comment:
This doesn't have a default arg, but I guess people shouldn't really call this function directly anyway

Reply:
This is the .cpp file. The .h file has the default arg so I guess I can mark this resolved.